mybatis-plus/src/main/java/com/baeldung/mybatisplus/MyBatisPlusConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.mybatisplus;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.baomidou.mybatisplus.annotation.DbType;
+import com.baomidou.mybatisplus.extension.plugins.MybatisPlusInterceptor;
+import com.baomidou.mybatisplus.extension.plugins.inner.PaginationInnerInterceptor;
+
+@Configuration
+public class MyBatisPlusConfig {
+
+    @Bean
+    public MybatisPlusInterceptor mybatisPlusInterceptor() {
+        MybatisPlusInterceptor interceptor = new MybatisPlusInterceptor();
+        interceptor.addInnerInterceptor(new PaginationInnerInterceptor(DbType.H2));
+        //interceptor.addInnerInterceptor(new PaginationInnerInterceptor(DbType.MYSQL));
```

Comment:
Don't leave commented out lines in the PR
```suggestion
```

Reply:
fixed